core/templates/dev/head/services/AudioPlayerService.js

```diff
@@ -81,6 +96,19 @@ oppia.factory('AudioPlayerService', [
       },
       rewind: function(seconds) {
         _rewind(seconds);
+      },
+      isPlaying: function() {
+        if (!_currentTrack) {
+          return false;
+        }
+        return !_currentTrack.paused;
+      },
+      trackLoaded: function() {
```

Comment:
start with verb. (This one is particularly misleading because "track" is a verb too :P)